pkg/pipelineascode/match.go

```diff
@@ -138,16 +142,23 @@ is that what you want? make sure you use -n when generating the secret, eg: echo
 		RemoteTasks:  p.run.Info.Pac.RemoteTasks,
 	})
 	if err != nil && !strings.Contains(err.Error(), "404 Not Found") {
+		p.eventEmitter.EmitMessage(repo, zap.ErrorLevel, err.Error())
 		return nil, nil, err
 	}
 	if pipelineRuns == nil {
 		msg := fmt.Sprintf("cannot locate templates in %s/ directory for this repository in %s", tektonDir, p.event.HeadBranch)
-		p.logger.Info(msg)
+		if err != nil {
+			msg += fmt.Sprintf(" err: %s", err.Error())
+		}
+		p.eventEmitter.EmitMessage(nil, zap.InfoLevel, msg)
 		return nil, nil, nil
 	}
 
 	// if /test command is used then filter out the pipelinerun
 	pipelineRuns = filterRunningPipelineRunOnTargetTest(p.event.TargetTestPipelineRun, pipelineRuns)
+	if p.event.TargetTestPipelineRun != "" && pipelineRuns == nil {
```

Comment:
is that a different fix? what was the bug ? and maybe a  test for it?

Reply:
it wasn't required ... having it also doesn't change anything :)
removed it.